packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -15,44 +15,47 @@ import { packagesToRemove } from './updates-for-v8/replaced-packages';
-    ...(options.enableCustomization ? ['@o3r/components', '@o3r/configuration'] : []),
-    ...(options.enableAnalytics ? ['@o3r/analytics'] : []),
-    ...(options.enableApisManager ? ['@o3r/apis-manager'] : []),
-    ...(options.enableStorybook ? ['@o3r/storybook'] : []),
-    ...(options.enablePlaywright ? ['@o3r/testing'] : []),
-    ...(options.enableRulesEngine ? ['@o3r/rules-engine'] : [])
-  ]));
-  const externalPackagesToInstallWithNgAdd = Array.from(new Set([
-    ...(options.enablePrefetchBuilder ? ['@o3r/ngx-prefetch'] : [])
-  ]));
-  return chain([
-    o3rBasicUpdates(options.projectName, o3rCoreVersion),
-    updateImports(mapImportV7toV8, renamedPackagesV7toV8),
-    updateBuildersNamesFromV7(),
-    updateOtterGeneratorsNamesFromV7(),
-    options.enableCms ? updateCmsAdapter(options, __dirname) : noop,
-    updateOtterEnvironmentAdapter(options, __dirname),
-    updateStore(options, __dirname),
-    updateFixtureConfig(options, __dirname),
-    options.enableCustomization ? updateCustomizationEnvironment(__dirname, o3rCoreVersion, options) : noop,
-    options.generateAzurePipeline ? createAzurePipeline(options, __dirname) : noop,
-    options.enablePlaywright ? updatePlaywright(__dirname) : noop,
-    updateLinter(options, __dirname, o3rCoreVersion),
-    updateAdditionalModules(options, __dirname),
-    generateRenovateConfig(__dirname),
-    removePackages(packagesToRemove),
-    options.skipLinter ? noop() : applyEsLintFix(),
-    // dependencies for store (mainly ngrx, store dev tools, storage sync), playwright, linter are installed by hand if the option is active
-    options.skipInstall ? noop() : install,
-    ngAddPackages(internalPackagesToInstallWithNgAdd, {skipConfirmation: true, version: o3rCoreVersion, parentPackageInfo: '@o3r/core - setup', projectName: options.projectName}),
-    ngAddPackages(externalPackagesToInstallWithNgAdd, {skipConfirmation: true, parentPackageInfo: '@o3r/core - setup', projectName: options.projectName})
-  ]);
+    const internalPackagesToInstallWithNgAdd = Array.from(new Set([
+      ...(workspaceProject.projectType === 'application' ? ['@o3r/application'] : []),
+      ...(options.enableCms ? ['@o3r/localization', '@o3r/styling', '@o3r/components', '@o3r/configuration'] : []),
+      ...(options.enableStyling ? ['@o3r/styling'] : []),
+      ...(options.enableConfiguration ? ['@o3r/configuration'] : []),
+      ...(options.enableLocalization ? ['@o3r/localization'] : []),
+      ...(options.enableCustomization ? ['@o3r/components', '@o3r/configuration'] : []),
+      ...(options.enableAnalytics ? ['@o3r/analytics'] : []),
+      ...(options.enableApisManager ? ['@o3r/apis-manager'] : []),
+      ...(options.enableStorybook ? ['@o3r/storybook'] : []),
+      ...(options.enablePlaywright ? ['@o3r/testing'] : []),
+      ...(options.enableRulesEngine ? ['@o3r/rules-engine'] : [])
+    ]));
+    const externalPackagesToInstallWithNgAdd = Array.from(new Set([
```

Comment:
I think you should add `@schematics/angular` to the list as you need it in your module `ng add`.
Because you are correctly importing it as dynImport but in case of failure to request to execute the core's `ng add` which does not seem to bring this dep to your app

Reply:
This is not new and we already discussed of making it a bugfix. 
Today there is an issue that ``@schematics/angular`` is a dependency of ``@o3r\schematics`` which is the base for all our schematics. I am currently investigating the issue on a different branch